koku/masu/database/presto_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -277,6 +297,10 @@ SELECT aws.uuid as aws_uuid,
         max(nullif(aws.lineitem_currencycode, '')) as currency_code,
         max(aws.lineitem_unblendedcost) as unblended_cost,
         max(aws.lineitem_unblendedcost) * cast({{markup}} as decimal(24,9)) as markup_cost,
+        max(aws.lineitem_blendedcost) as blended_cost,
+        max(aws.lineitem_blendedcost) * cast({{markup}} as decimal(33,15)) as markup_cost_blended,
+        max(savingsplan_savingsplaneffectivecost) as savingsplan_effective_cost,
+        max(savingsplan_savingsplaneffectivecost) * cast({{markup}} as decimal(33,15)) as markup_cost_savingsplan,
```

Comment:
nitpick (non-blocking): we prefixed all of the other elements with `aws.` or `ocp.` to identify where they came from even if they aren't ambiguous, if we wanted to add that here we could